### DISCUSSION TITLE
Wazuh agent error

### DISCUSSION THREAD

**Discussion by usmanaa10** [Security-Onion-Solutions/securityonion#10967] [Unsupported Versions]
Hi fellows,
          I am facing a problem, my agents are not connecting with serving giving an error  i.e. unable to connect x.x.x.x:1515, the steps i have done are as follow 
On Server:
1) Check wazuh manager version
2) Added agent's ip
3) so-allow agent ip

ON Endpoints

1) Install the same version of the agent

error message
"

C:\Program Files (x86)\ossec-agent>agent-auth.exe -m  X.X.X.X
2023/08/07 15:28:36 agent-auth: INFO: Started (pid: 3616).
2023/08/07 15:28:37 agent-auth: INFO: Starting enrollment process to server: X.X.X.X
2023/08/07 15:28:58 agent-auth: ERROR: Unable to connect to X.X.X.X:1515

"

**Comment by cm-ops** [comment]
Can you share a little more about your grid? What type of node are you trying to connect to (manager, sensor)? If you run `iptables -nL | grep 1515` do you see your firewall accept?

**Reply by usmanaa10** [reply]
Hi @cm-ops 

Here is the output of the suggested command(iptables -nL | grep 1515).
 
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/66419518/899f24a1-443d-4d33-9e3f-963bd16a3fd6)

**Grid Detail**
 
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/66419518/3100c449-e757-402f-a7d1-23bf4bfa691e)

I am trying to add a window agent from 192.168.X.0/24 block (**Agent and server are on same VLAN**) to the server and getting an error of unable to connect on port 1515.

I also allow ports 1515-1514 TCP and UDP on the firewall of the endpoint.

And also tested after turning off the endpoint firewall but it doesn’t work. 

Yes when I telnet my SO IP with port 1515 or 1514 it doesn’t make a connection but when I telnet server IP with 80 it works, but when I netstat I see 1515 and 1514 in the listening state
